components/SearchBar/index.js

```diff
@@ -86,19 +90,21 @@ function SearchBar({ search, onSearch }) {
           </Styled.FilterContainer>
           <Styled.FilterContainer>
             <Styled.FilterItem>
-              <label>Date Start</label>
+              <label for="date_start">Date Start</label>
```

Comment:
[We need to use `htmlFor` in React](https://legacy.reactjs.org/docs/dom-elements.html#htmlfor).